detectors/node/opentelemetry-resource-detector-docker/README.md

```diff
@@ -0,0 +1,48 @@
+# OpenTelemetry Resource Detector for Docker
+
+[![NPM Published Version][npm-img]][npm-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
+
+[component owners](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/.github/component_owners.yml): @abhee11
+
+Resource detector for docker.
+
+Compatible with OpenTelemetry JS API and SDK `1.0+`.
+
+## Installation
+
+```bash
+npm install --save @opentelemetry/resource-detector-docker
+```
+
+## Usage
+
+```typescript
+import { detectResources } from '@opentelemetry/resources';
+import { dockerCGroupV1Detector } from '@opentelemetry/resource-detector-docker'
+const resource = await detectResources({
+   detectors: [dockerCGroupV1Detector],
+})
+
+const tracerProvider = new NodeTracerProvider({ resource });
+```
+
+## Available detectors
```

Comment:
A detector per attribute? Seems like it would be better to have a `DockerDetector` and list the attributes it detects here

Reply:
@dyladan thank you for the comments
Currently I have named it as DockerCGroupV1Detector given it is specific to CGroup v1 
We currently just have container ID that is added as a part of resouce, In future we could definitely add more attributes but I think it is important to mark the name CGroupV1 

Do you agree ?

Reply:
OK yeah thats fine